server/api/collections.js

```diff
@@ -562,9 +563,17 @@ router.post("collections.update", auth(), async (ctx) => {
     collection.color = color;
   }
   if (permission !== undefined) {
+    // frontend sends empty string
+    ctx.assertIn(
+      permission,
+      ["read_write", "read", "", null],
```

Comment:
```suggestion
      ["read_write", "read", ""],
```

Can it really be `null`?

Reply:
Some tests pass `null` I didn't update them with `""` thinking it must be some legacy feat that we are taking care of.